app/components/candidate_interface/rejection_reasons/reasons_for_rejection_component.html.erb

```diff
@@ -0,0 +1,183 @@
+<% if reasons.candidate_behaviour_y_n == 'Yes' %>
```

Comment:
Is it not possible to somehow use the methods in `ReasonsForRejectionPresenter`?

Reply:
Possibly. I haven't done any work on this and I'm not going to think about refactoring it because it's a legacy format.